mungegithub/mungers/lgtm_handler.go

```diff
@@ -81,39 +81,51 @@ func (h *LGTMHandler) addLGTMIfCommented(obj *github.MungeObject, comments []*gi
 	// Assumption: The comments should be sorted (by default from github api) from oldest to latest
 	for i := len(comments) - 1; i >= 0; i-- {
 		comment := comments[i]
-		fields := strings.Fields(*comment.Body)
+		if !mungerutil.IsValidUser(comment.User) {
+			continue
+		}
+
+		if !(mungerutil.IsMungeBot(comment.User) || isReviewer(comment.User, reviewers)) {
+			continue
+		}
+
+		fields := getFields(*comment.Body)
 		if isCancelComment(fields) {
-			// "/lgtm cancell" if commented more recently than "/lgtm"
+			// "/lgtm cancel" if commented more recently than "/lgtm"
 			return
 		}
 		if isLGTMComment(fields) {
```

Comment:
Actually what if we made the lgtm_after_commit handler do it, rather than do it here. He if change /lgtm into something else (with a comment why it changed it)